explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -179,41 +195,39 @@ object AladinCell extends ModelOptics with AladinCommon:
     ScalaFnComponent
       .withHooks[Props]
       .useContext(AppContext.ctx)
-      // mouse coordinates, starts on the base
-      .useStateBy((props, _) => props.asterism.baseTracking.baseCoordinates)
       // target options, will be read from the user preferences
       .useStateViewWithReuse(Pot.pending[(UserGlobalPreferences, TargetVisualOptions)])
       // to get faster reusability use a serial state, rather than check every candidate
       .useSerialState(List.empty[GuideStarCandidate])
       // Analysis results
       .useSerialState(List.empty[AgsAnalysis])
-      // Ags state
-      .useState[AgsState](AgsState.Idle)
       // Request data again if vizTime changes more than a month
-      .useEffectWithDepsBy((p, _, _, _, _, _, _) => p.obsConf.vizTime) {
-        (props, ctx, _, _, gs, _, agsState) => vizTime =>
+      .useEffectWithDepsBy((p, _, __, _, _) => p.obsConf.vizTime) {
```

Comment:
Is this two `_` side by side? I didn't know that would work.

Reply:
oh weird, it certainly compiles